lib/assessment.js

```diff
@@ -8,14 +8,23 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
+var min = function min(callback, myArray) {
+
+  return myArray.reduce(function (p, v) {
+    return ( p < v ? p : v );
+  });
+}
+
```

Comment:
Good answer, but wrong format. The question asked for the **callback** that we'd pass into reduce. 
e.g.
```javascript
var cb = function(p, v){
...
};
[1,2,3].reduce(cb);
```